corehq/apps/userreports/models.py

```diff
@@ -151,9 +151,17 @@ def indicators(self):
                 }
             }
         }, self.named_filter_objects)
+        repeat_iteration_indicator = IndicatorFactory.from_spec({
+            "column_id": "repeat_iteration",
+            "type": "repeat_iteration",
+            "display_name": "base document iteration",
+            "datatype": "integer",
+            "is_nullable": False,
+            "is_primary_key": True,
```

Comment:
I don't think they will change. I've gotten rid of the spec altogether, but I could also set defaults on all the values (like in 21cb915).